pull--acs5-general-data.R

```diff
@@ -903,7 +903,7 @@ acs_incpov <-
 # Prepare income-to-poverty ratio variables.
 
 # /!\ NSM: Easier if data were reshaped?
-for (r in c("r0to49", "r50to74", "r75to99", "r100to124", "r125to149", "r150to174", "r175to184", 
+for (r in c("r0to50", "r50to74", "r75to99", "r100to124", "r125to149", "r150to174", "r175to184", 
```

Comment:
What was the rationale for going from 0to49 to 0to50? My recollection is hazy, but my sense was that you were the one that established 0to49 as a more consistent naming...